src/Livewire/Product/Product.php

```diff
@@ -58,13 +71,21 @@ public function mount(int $id): void
         $product->append('avatar_url');
 
         $this->product->fill($product);
+        $this->product->product_properties = Arr::keyBy($this->product->product_properties, 'id');
 
         $this->additionalColumns = $product->getAdditionalColumns()->toArray();
+        $this->recalculateDisplayedProductProperties();
+
+        $this->view = data_get(
+            ProductType::get($product->product_type) ?? ProductType::getDefault(),
+            'view',
+            $this->view
+        );
     }
 
     public function render(): View|Factory|Application
     {
-        return view('flux::livewire.product.product', [
+        return view($this->view, [
```

Comment:
have a look at https://laravel.com/docs/11.x/views#creating-the-first-available-view
i think this can be used here… just a hint, works your way too…